rules/manager.go

```diff
@@ -264,16 +264,23 @@ type Group struct {
 	logger log.Logger
 
 	metrics *Metrics
+
+	ruleGroupPostProcessFunc RuleGroupPostProcessFuncType
 }
 
+// This function will be used before each rule group evaluation if not nil.
+// Use this function type if the rule group post processing is needed.
+type RuleGroupPostProcessFuncType func(g *Group, lastEvalTimestamp time.Time, log log.Logger) error
```

Comment:
```suggestion
type RuleGroupPostProcessFunc func(g *Group, lastEvalTimestamp time.Time, log log.Logger) error
```

To be consistent with the rest of the codebase.